pkg/controller/node/nodecontroller.go

```diff
@@ -955,7 +954,16 @@ func (nc *NodeController) tryUpdateNodeStatus(node *v1.Node) (time.Duration, v1.
 		}
 
 		// remaining node conditions should also be set to Unknown
-		remainingNodeConditionTypes := []v1.NodeConditionType{v1.NodeOutOfDisk, v1.NodeMemoryPressure, v1.NodeDiskPressure}
+		remainingNodeConditionTypes := []v1.NodeConditionType{
+			v1.NodeOutOfDisk,
+			v1.NodeMemoryPressure,
+			v1.NodeDiskPressure,
+			// We don't change 'NodeInodePressure' condition, as it'll be removed in future.
```

Comment:
Good point! Opened https://github.com/kubernetes/kubernetes/issues/49103 to confirm whether we can remove it in 1.8 :).